calendarx.d.ts

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+
+export type DateLike = Date | string | number
+
+export interface Event {
+    date: Date;

```

Comment:
If I recall correctly you can also specify `startDate` and `endDate` right? Maybe something like this would work:
```
export type Event = { date: DateLike } | { startDate: DateLike, endDate: DateLike }
```